Doc/library/typing.rst

```diff
@@ -1388,12 +1388,20 @@ These are not used in annotations. They are building blocks for declaring types.
    ``Point2D.__total__``, ``Point2D.__required_keys__``, and
    ``Point2D.__optional_keys__``.
    To allow using this feature with older versions of Python that do not
-   support :pep:`526`, ``TypedDict`` supports two additional equivalent
-   syntactic forms::
+   support :pep:`526`, ``TypedDict`` can be created using a functional form:
```

Comment:
I'm not keen on the repetition of the word "using" here, and with the bullet points I think we can go back to something similar to the existing text:

```suggestion
   support :pep:`526`, ``TypedDict`` supports two additional
   syntactic forms::
```